src/tests/example.spec.js

```diff
@@ -28,3 +27,154 @@ test.describe('Saucedemo app basic tests', () => {
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
 });
+
+
+
+test.describe('Saucedemo UNIT 10', () => {
+        test.beforeEach( async (/** @type {{ app: import('../pages/Application').Application }} */{ app },) =>{
+            await app.login.navigate();
+            await app.login.performLogin('standard_user', 'secret_sauce');
+        } );
+
+
+        test('1. Check sorting by Name A Z', async ({app})=> {
+                  await app.inventory.sortField.selectOption('az');
+                  const goodsName = await app.inventory.inventoryItemsName.allTextContents();
+                  await expect([...goodsName].sort()).toEqual(goodsName);
+          });
+
+        test('1. Check sorting by Name Z A', async ({app})=> {
+                await app.inventory.sortField.selectOption('za');
+                const goodsName = await app.inventory.inventoryItemsName.allTextContents();
+ //             await expect([...goodsName].sort().reverse()).toEqual(goodsName);
```

Comment:
Why is this line commented out? Does it work?

Reply:
Yup ) it works) just another way to do same thinf